### PR TITLE
Encode key registration transactions over REST API

### PR DESCRIPTION
**Pull request by tsachiherman**
## Summary

Encode key registration transactions over REST API.

## Notes

This is a modified implementation of the logging mechanism used in 
https://github.com/algorand/go-algorand/pull/167

If approved, the goal is to accept this one and close both.

### CODE REVIEW

File: daemon/algod/api/server/v1/handlers/errors.go [hunk 1]
```diff
@@ -17,17 +17,21 @@
 package handlers
 
 var (
```
Comment (tsachiherman) [inline daemon/algod/api/server/v1/handlers/errors.go]: The changes here aren't that substantial; I just reorder the errors alphabetically while adding three.

**Comment by tsachiherman** [comment]
I'm not sure if @mxmauro has a reviewer permissions here, but would be happy to have his feedback.

**Comment by Vervious** [comment]
Looks like we want to either merge this PR, or merge #167 with Tsachi's changes pulled in. Up to @mxmauro which way is preferable

**Comment by Vervious** [comment]
Hi, I accidentally squashed your commits so lost authorship recognition. I will rectify this somehow @mxmauro @scnale

**Comment by mxmauro** [comment]
Don't worry @Vervious . I'll wait for the release to index those txs too :)

**Comment by Vervious** [comment]
#210

**Pull request merged**